src/reporting/Branch.sol

```diff
@@ -78,8 +78,9 @@ contract Branch is DelegationTarget, Typed, Initializable, IBranch {
         return reportingWindows[_reportingWindowId];
     }
 
-    // TODO: this has a signature conflict with the public version below.  for consistency, we should have this be the signature for the `constant` version and rename the `public` one
-    //function getChildBranch(bytes32 _parentPayoutDistributionHash) constant returns (Branch);
+    function getChildBranch(bytes32 _parentPayoutDistributionHash) constant returns (IBranch) {
```

Comment:
This method (and many others in this contract) needs a visibility modifier.  `public` seems reasonable.